spec/models/team_spec.rb

```diff
@@ -41,14 +41,12 @@
   end
 
   describe 'team settings' do
-    # allow_self_leaveがデフォルト値としてtrueを返す
-    it 'has default value for allow_self_leave' do
-      team = described_class.new
+    it '自己退会の設定はデフォルト値としてtrueを返す' do
+      team = Team.new
       expect(team.allow_self_leave).to be true
```

Comment:
```suggestion
      expect(team.allow_self_leave?).to be true
```

true/falseを確認したいときは?を付けることが多いかなー